llvm/test/CodeGen/RISCV/rvv/fixed-vectors-strided-load-combine.ll

```diff
@@ -9,9 +9,9 @@
 define void @widen_2xv4i16(ptr %x, ptr %z) {
 ; CHECK-LABEL: widen_2xv4i16:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    vsetivli zero, 8, e16, m1, ta, ma
-; CHECK-NEXT:    vle16.v v8, (a0)
-; CHECK-NEXT:    vse16.v v8, (a1)
+; CHECK-NEXT:    vsetivli zero, 2, e64, m1, ta, ma
```

Comment:
This may cause unaligned access error?

Reply:
The combine checks for the alignment with `TLI.isLegalStridedLoadStore(WideVecVT, Align)`, so I think this should be fine. This is what it was originally combining to anyway, the generic dag combine was just re-combining it smaller SEW

Reply:
So the alignment of `<4 x i16>` is 64 bits, IIUC? Then this makes sense to me.